spec/real_examples/cooking_nytimes_spec.rb

```diff
@@ -9,8 +9,8 @@
 
     subject { @parsed }
 
-    it "should use a non-standard parser" do
-      expect(Hangry::ParserSelector.new(@html).parser).to be_an_instance_of(Hangry::Parsers::NonStandard::NyTimesParser)
+    it "should use the schema.org parser" do
+      expect(Hangry::ParserSelector.new(@html).parser).to be_an_instance_of(Hangry::SchemaOrgRecipeParser)
```

Comment:
I would be inclined to get rid of all these tests too, as per your point about sites changing their schemas.